src/pages/ProfileViewPage.tsx

```
@@ -0,0 +1,62 @@
+import { NextProfile, PageDots, ProfileCard, ProgressBar } from "@/components/profileview";
+import type { Profile } from "@/components/profileview/types";
```

ESLint

[error] 'Profile' is defined but never used.

(@typescript-eslint/no-unused-vars)